<!--xml -->

<launch>
	<node pkg="remote_transport" type="msg_sender_node" name="msg_sender_node" output="screen">
		<param name="image_topic" value="/image_raw"/>
		<param name="socket_ip" value="47.98.51.23"/>
		<param name="socket_port" value="8888"/>
	</node>
</launch>
